code/game/gamemodes/dynamic/dynamic_rulesets.dm

```diff
@@ -170,8 +173,12 @@
 /// Here you can perform any additional checks you want. (such as checking the map etc)
 /// Remember that on roundstart no one knows what their job is at this point.
 /// IMPORTANT: If ready() returns TRUE, that means pre_execute() or execute() should never fail!
-/datum/dynamic_ruleset/proc/ready(forced = 0)
-	return check_candidates()
```

Comment:
Hmmmm are we sure that forcing everything to call this doesn't change behavior at all? I am fairly sure some of the calls without were intentional

Reply:
how about just loading the templates before it calls ready?

Reply:
Yeah